src/ssl.c

```diff
@@ -14307,7 +14316,15 @@ WOLFSSL_X509* wolfSSL_X509_d2i(WOLFSSL_X509** x509, const byte* in, int len)
 
     return newX509;
 }
-
+#ifndef NO_WOLFSSL_STUB
+WOLFSSL_X509* wolfSSL_d2i_X509_fp(FILE *fp, WOLFSSL_X509 **x509)
```

Comment:
wolfSSL_d2i_X509_fp() should be excluded when NO_FILESYSTEM is defined, as FILE is not available.

Reply:
I added #ifdef NO_FILESYSTEM.